src/constants/sfx.asm

```diff
@@ -2,278 +2,278 @@
-MUSIC_MARIN_BEACH                               equ $4E
-;MUSIC_UNKNOWN                                  equ $4F ; Jingle, possible unused/possibly related to Marin
-MUSIC_MINIBOSS                                  equ $50
-MUSIC_KANALET_CASTLE_COPY                       equ $51 ; Copy of $13.
-MUSIC_TAIL_CAVE_COPY                            equ $52 ; Copy of $14.
-MUSIC_DREAM_SHRINE                              equ $53 ; Plays in the lower Dream Shrine. (After sleeping in the bed)
-MUSIC_EAGLES_TOWER_BOSS_CUTSCENE                equ $54 ; Plays when the boss comes into view. Switches to $19.
-MUSIC_ROOSTER_REVIVAL                           equ $55 ; Plays while the flying rooster's spirit reenters his remains.
-MUSIC_SEASHELL_MANSION_SPIRIT                   equ $56 ; Plays $55 afterwards. Possibly left unused for resurrecting the rooster?
-MUSIC_CUCCO_HOUSE                               equ $57
-MUSIC_FACE_SHRINE                               equ $58
-MUSIC_MEETING_WINDFISH                          equ $59 ; When you climb the staircase and meet the Wind Fish.
-MUSIC_TURTLE_ROCK                               equ $5A
-MUSIC_EAGLE_TOWER                               equ $5B
-MUSIC_GRIM_CREEPER_DIALOG                       equ $5C ; Plays during Grim Creeper's dialog before the boss. Switches to $54 after the dialog.
-MUSIC_FINAL_BOSS_DIALOG                         equ $5D ; Plays during the final boss' dialog.
-MUSIC_BOSS_WARNING                              equ $5E ; Plays during the dialogs where the bosses tell you not to wake the Wind Fish.
-MUSIC_FINAL_BOSS_DEFEATED                       equ $5F
-MUSIC_ZELDA_NICKNAME_EASTER_EGG                 equ $60 ; ZELDA nickname easter egg.
-MUSIC_COLOR_DUNGEON                             equ $61
-;MUSIC_COLOR_DUNGEON_COPIES                     equ $62 to 6F   ; Same as $61
-;MUSIC_GLITCHED                                 equ $70 to $FE  ; Glitches the music that played before it.
-MUSIC_SILENCE                                   equ $FF ; Seems to try to silence the voice channels. Does not always correctly silence the music.
+MUSIC_NONE                              equ $00
+MUSIC_TITLE_SCREEN_INTRO                equ $01
+MUSIC_MINIGAME                          equ $02 ; Trendy Game, Fishing, Camera Shop
+MUSIC_GAME_OVER                         equ $03
+MUSIC_MABE_VILLAGE                      equ $04
+MUSIC_OVERWORLD                         equ $05
+MUSIC_TAL_TAL_RANGE                     equ $06
+MUSIC_SHOP                              equ $07 ; Town Shop, Witch's Hut
+MUSIC_RAFT_RIDE_RAPIDS                  equ $08
+MUSIC_MYSTERIOUS_FOREST                 equ $09
+MUSIC_TOWN_HOUSE                        equ $0A
+MUSIC_ANIMAL_VILLAGE                    equ $0B
+MUSIC_FAIRY_FOUNTAIN                    equ $0C
+MUSIC_TITLE_SCREEN                      equ $0D ; If intro skipped with Start
+MUSIC_BOWWOW_KIDNAPPED                  equ $0E
```

Comment:
The old comment here referred to `$3E` (ie. `MUSIC_BOWWOW_KIDNAPPED_INTRODUCTION`) which is probably useful to know about